assets/stylesheets/environment/tools/mixins/_grid-layout.scss

```diff
@@ -26,12 +26,13 @@
 ///   div {
 ///     @include grid-column(1 / 2, $float: right);
 ///   }
-@mixin grid-column($width, $full-width: tablet, $float: left) {
+@mixin grid-column($width, $full-width: desktop, $float: left) {
   padding: 0 ($gutter / 2);
+  max-width: $measure;
```

Comment:
I'd say this isn't the right place because doesn't this prevent the column flowing full width of the page?